GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -262,106 +259,60 @@ private void OnGuiLoadingScreenSelect()
                 EditorGUILayout.TextField(Config.AssetBundleUrl, GUILayout.MinWidth(FieldMinWidth));
             EditorGUILayout.EndHorizontal();
 
-            EditorGUILayout.Space();
-
-            var setAssetBundleText = QuickDeployConfig.EngineConfigExists()
-                ? "Update AssetBundle URL"
-                : "Set AssetBundle URL";
-
-            if (GUILayout.Button(setAssetBundleText))
-            {
-                try
-                {
-                    Config.SaveConfiguration(ToolBarSelectedButton.LoadingScreen);
-                }
-                catch (Exception ex)
-                {
-                    DialogHelper.DisplayMessage(AssetBundleCheckerErrorTitle, ex.Message);
-
-                    throw;
-                }
-            }
-
-            EditorGUILayout.Space();
-            if (GUILayout.Button("Check AssetBundle"))
-            {
-                var window = AssetBundleVerifierWindow.ShowWindow();
-
-                try
-                {
-                    Config.SaveConfiguration(ToolBarSelectedButton.LoadingScreen);
-                    window.StartAssetBundleDownload(Config.AssetBundleUrl);
-                }
-                catch (Exception ex)
-                {
-                    DialogHelper.DisplayMessage(AssetBundleCheckerErrorTitle, ex.Message);
-
-                    window.Close();
-
-                    throw;
-                }
-            }
-
-            EditorGUILayout.Space();
-
-            EditorGUILayout.EndVertical();
-            EditorGUILayout.Space();
-
-            EditorGUILayout.Space();
-            EditorGUILayout.LabelField("Select Loading Screen Image", EditorStyles.boldLabel);
-            EditorGUILayout.BeginVertical(UserInputGuiStyle);
-            EditorGUILayout.Space();
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider); // Draws a horizontal line
```

Comment:
This doesn't really look like other UI that we have. Should we instead do separate shaded sections? Or just combine them outright?

Reply:
I'll try both and we can see which we prefer.

Reply:
Resolved